lib/matplotlib/patches.py

```diff
@@ -1559,14 +1559,15 @@ def get_radius(self):
 
 class Arc(Ellipse):
     """
-    An elliptical arc.  Because it performs various optimizations, it
-    can not be filled.
-
-    The arc must be used in an :class:`~matplotlib.axes.Axes`
-    instance---it can not be added directly to a
-    :class:`~matplotlib.figure.Figure`---because it is optimized to
-    only render the segments that are inside the axes bounding box
-    with high resolution.
+    An elliptical arc, i.e. a segment of an ellipse.
+
+    Due to internal optimizations, there are certain restrictions on using Arc:
+
+    - The arc cannot be filled.
+
+    - The arc must be used in an :class:`~.axes.Axes` instance---it can not be
```

Comment:
If you add spaces around `---`, would it turn into a real em-dash?

Reply:
No.

![grafik](https://user-images.githubusercontent.com/2836374/47971781-f2515c00-e094-11e8-89e9-61ba69d97483.png)